routers/likes.py

```
@@ -1,31 +1,142 @@
+
+    return {"message": "Liked successfully"}
+
+
+@router.delete("/posts/{post_id}/likes", status_code=status.HTTP_204_NO_CONTENT)
+def delete_like(post_id: PostId, user_id: CurrentUserId):
+    """좋아요 취소"""
+    _, posts = _get_post_or_404(post_id)
+    likes = read_json(settings.likes_file)
+
+    like_index = next(
+        (i for i, l in enumerate(likes) if l["post_id"] == post_id and l["user_id"] == user_id),
```

Ruff (0.14.13)

Ambiguous variable name: `l`

(E741)